classes/PodsAPI.php

```diff
@@ -4201,6 +4201,18 @@ public function duplicate_pod_item ( $params ) {
         );
 
         foreach ( $fields as $field ) {
+	        if ( in_array( $field['name'], array(
+		        'ID',
+		        'post_name',
+		        'post_date',
```

Comment:
I think we can remove post_date, post_date_gmt, post_modified, post_modified_gmt from this list.